other_procore_apis/bim_web_viewer.md

```diff
@@ -832,13 +791,541 @@ Manually dispatches an event
+
+---
+
+### objectUnhide
+
+<p class="heading-link-container"><a class="heading-link" href="#objectunhide"></a></p>
+
+Fires when objects have been removed from the hidden set.
+
+#### Data Properties
+
+| Field Name | Type   | Description                                                                                               |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------- |
+| origin     | string | The origin of the un hide operation, can be: `coachmark_hidden`, `object_tree`, `object_tree_all_objects` |
+
+---
+
+### objectSelect
+
+<p class="heading-link-container"><a class="heading-link" href="#objectselect"></a></p>
+
+Fires when an object has been selected. This event returns Meshnode ID.
+
+---
+
+### optionsChanged
+
+<p class="heading-link-container"><a class="heading-link" href="#optionschanged"></a></p>
+
+Fires when the Webviewer options have changed. This event fires when `setOptions` is called, or when actions cause the multiple selection option to change, such as keyboard shortcuts to multi select objects.
+
+---
+
+### objectMeasure
+
+<p class="heading-link-container"><a class="heading-link" href="#objectmeasure"></a></p>
+
+Fires objects are selected for measurement. This event returns a Meshnode ID of the object selected fore measurement.
+
+---
+
+### renderReady
+
+<p class="heading-link-container"><a class="heading-link" href="#renderready"></a></p>
+
+Fires when the webviewer is ready to render the model.
+
+#### Data Properties
+
+| Field Name   | Type   | Description                                    |
+| ------------ | ------ | ---------------------------------------------- |
+| offsetHeight | number | Offset height of the parent element container. |
+| offsetLeft   | object | Offset left of the parent element container.   |
+| offsetTop    | number | Offset top of the parent element container.    |
+| offsetWidth  | object | Offset width of the parent element container.  |
+
+---
+
+### redo
+
+<p class="heading-link-container"><a class="heading-link" href="#redo"></a></p>
+
+Fires when the keyboard shortcut to redo is pressed. This event fires regardless if there is anything to redo.
+
+---
+
+### selectTool
+
+<p class="heading-link-container"><a class="heading-link" href="#selecttool"></a></p>
+
+Fires when a tool has been activated.
+
+#### Data Properties
+
+| Field Name | Type   | Description                                                                                 |
+| ---------- | ------ | ------------------------------------------------------------------------------------------- |
+| origin     | string | The origin of the tool being activated.                                                     |
+| tool       | number | The tool that has been activated, enumerated from `ProcoreBim.Webviewer.toolstoolbar_enum`. |
```

Comment:
Looks like this should be `toolbar_enum`.

![image](https://user-images.githubusercontent.com/2865469/186540650-c03df496-c26b-41f1-81ad-27b8600ce135.png)


```suggestion
| tool       | number | The tool that has been activated, enumerated from `ProcoreBim.Webviewer.toolbar_enum`. |
```